tools/experiments/experiments.js

```diff
@@ -138,6 +143,34 @@ function build() {
   } else {
     redirect.setAttribute('hidden', '');
   }
+
+  const rtvInput = document.getElementById('rtv');
+  const rtvButton = document.getElementById('rtv-submit');
+  rtvInput.addEventListener('input', () => {
+    rtvButton.disabled = !(
```

Comment:
Could you use `<input pattern="^(\\d{15})?$">` here instead of implementing this test in a listener?

Reply:
I believe this only takes effect if the <input> is inside a <form> that is being submitted? In any case, I use the pattern for other decisions so I would still need to define it in the JS